exercises/crypto-square/README.md

```diff
@@ -47,9 +47,10 @@ imtgdvsfearwermayoogoanouuiontnnlvtwttddesaohghnsseoau
 
 Output the encoded text in chunks.  Phrases that fill perfect rectangles
 `(r X c)` should be output `c` chunks of `r` length, separated by spaces.
-Phrases that do not fill perfect rectangles will have `n` empty spaces.
-Those spaces should be distributed evenly, added to the end of the last
-`n` chunks.
+Phrases that do not fill perfect rectangles will be `n` characters short.
```

Comment:
The Description.md and README.md needs to match for configlet to not throw an error. Shouldn't this mean that the [exercise specification](https://github.com/exercism/problem-specifications/blob/master/exercises/crypto-square/description.md) will need to be updated first?

Reply:
indeed. And this would be a good thing to do because (as http://x.exercism.io/problems/crypto-square tells me) the Rust track is not the only track that would benefit from a changed explanation. To share our work so that others can benefit, the change should be made to https://github.com/exercism/problem-specifications/blob/master/exercises/crypto-square/description.md first, before being made here.

Reply:
OK, I set off to fix the [exercise specification](https://github.com/exercism/problem-specifications/blob/master/exercises/crypto-square/description.md) and update [canonical-data.json](https://github.com/exercism/problem-specifications/blob/master/exercises/crypto-square/canonical-data.json) with my jaggedness tests.  I learned that the Rust implementation DOES NOT MATCH the specification!

Here's an item from the formal spec:
```json
{
  "description": "54 character plaintext results in 7 chunks, the last two with trailing spaces",
  "property": "ciphertext",
  "input": {
    "plaintext": "If man was meant to stay on the ground, god would have given us roots."
  },
  "expected": "imtgdvs fearwer mayoogo anouuio ntnnlvt wttddes aohghn  sseoau "
}
```

And here's the rust implementation of this test:
```rust
#[test]
fn test_example() {
    test(
        "If man was meant to stay on the ground, god would have given us roots.",
        "imtgdvs fearwer mayoogo anouuio ntnnlvt wttddes aohghn sseoau",
    )
}
```
Note that the "expected" outputs DIFFER -- specifically the extra spaces on the last two chunks are missing for the rust implementation.  This inconsistency is a big part of the problem this PR was supposed to fix.  I will clarify the problem description in the exercise spec, then, raise the inconsistency as a separate issue in the rust implementation.

Reply:
Here's my [exercism specification PR](https://github.com/exercism/problem-specifications/pull/1210).